### PR TITLE
first round of updates

### PR DESCRIPTION
**Pull request by amaseda**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,21 +2,20 @@
 
 # Concepts in Computer Science: Algorithms
 
-
 ## Objectives
```
Comment (nayana487) [inline README.md]: ## Learning Objectives

File: README.md [hunk 2]
```diff
@@ -2,21 +2,20 @@
...
 
-By the end of this, developers should be able to:
+By the end of this, developers should be able to...
 
 -   Define "algorithm"
```
Comment (nayana487) [inline README.md]: why `algorithm` in quotes?

File: README.md [hunk 3]
```diff
@@ -2,21 +2,20 @@
...
 -   Give an example of a divide-and-conquer algorithm
 -   Predict the time-complexity of a given algorithm
 
-## Computer Science (0:05, 5 min)
+## Computer Science (5 minutes, 0:05)
 
 How many of us find the concept or mention of computer science intimidating?
 
-Many think of  computer science as a necessary prerequisite to do anything programming-related with a computer and *really know what you are doing*, or that knowing a certain amount of computer science is some kind of litmus test for *true programmers*. This attitude perhaps unnecessarily mystifies an already difficult subject and field of study, and at worst, makes the learning curve seem so steep that it is like a learning barrier.
+Many think of computer science as a necessary prerequisite to do anything programming-related with a computer and *really know what you are doing*, or that knowing a certain amount of computer science is some kind of litmus test for *true programmers*. This attitude perhaps unnecessarily mystifies an already difficult subject and field of study, and at worst, makes the learning curve seem so steep that it is like a learning barrier.
```
Comment (nayana487) [inline README.md]: "for a true programmer"

Comment (nayana487) [reply README.md]: "This attitude perhaps unnecessarily mystifies an already difficult subject and field of study and, at worst, ..."

move the after `study`

File: README.md [hunk 4]
```diff
@@ -2,21 +2,20 @@
...
-By the end of this, developers should be able to:
+By the end of this, developers should be able to...
 
 -   Define "algorithm"
 -   Identify what Big-O time-complexity measures
 -   Give an example of a divide-and-conquer algorithm
 -   Predict the time-complexity of a given algorithm
 
-## Computer Science (0:05, 5 min)
+## Computer Science (5 minutes, 0:05)
```
Comment (nayana487) [inline README.md]: should we explicitly say what computer science is?

"Computer science is a discipline that involves the understanding and design of computers and computational processes. In its most general form it is concerned with the understanding of information transfer and transformation. Particular interest is placed on making processes efficient and endowing them with some form of intelligence. The discipline ranges from theoretical studies of algorithms to practical problems of implementation in terms of computational hardware and software..."

https://www.cs.mtu.edu/~john/whatiscs.html

File: README.md [hunk 5]
```diff
@@ -86,51 +85,51 @@ list:
...
 algorithms are better in different contexts, or with different constraints. It's
 up to you to consider the options and pick the one that best meets your needs.
 
 [Sorting Algorithms Visualized](https://www.toptal.com/developers/sorting-algorithms)
 
-### We do: Bubble Sort (5 min, 0:40)
+### We Do: Bubble Sort (5 minutes, 0:40)
 
-Next, we'll visualize bubble sorting with 7 volunteers. One person will play the role of recursive sorter.
+Next, we'll visualize bubble sorting with 7 volunteers. One person will play the role of the recursive sorter.
```
Comment (nayana487) [inline README.md]: so technically AP style is that if a number is less than 10 you spell it out....

(jk, trolling, but not really)

File: README.md (hunk 2 above)
Comment (amaseda) [reply README.md]: I dunno. I guess to indicate we're defining that specific term?

File: README.md (hunk 4 above)
Comment (amaseda) [reply README.md]: :+1:

File: README.md (hunk 5 above)
Comment (amaseda) [reply README.md]: I like it.

**Pull request merged**